doc/ref/gappkg.xml

```diff
@@ -1286,8 +1287,8 @@ This means that the binary is not regarded as essential for this
 package.
 <P/>
 You might also have to cope with the situation that external binaries will
-only run under UNIX (and not, say, under Windows), or may not compile with
-some compilers or default compiler options.
+only run under UNIX (and not, for example, under Windows), or may not compile
```

Comment:
You could also use `e.g.` here and in another few places as a replacement for `say` that has almost the same length and thus causes less of a ripple effect of other changes.

Reply:
I'm perfectly fine with either `e.g.` or `for example` but to make the endeavor maybe a bit smaller I'd suggest to leave the `say`s in situations where there is actually something after it like in this case. So every time it can be replaced with `for example` I'd leave it because using it that way is not that unusual. But in the places where it can just be removed using it is more unusual and also disturbs the flow.

Reply:
‘for example’ or ‘e.g.’ are both fine here, although I can’t quite work out the best way to put it in the parenthetical clause.

> and not e.g. under Windows

> and not under e.g. Windows

> and e.g. not under Windows

Sticking with the current ordering is probably best.

I don’t think the commas are needed in any case.